File: PHP/Easy/Filters/Math/412. Fizz Buzz.php
<?php
function fizzBuzz($n)
{
    $ans = [];
    for ($i = 1; $i <= $n; $i++) {
        if ($i % 3 == 0 and ($i % 5 == 0)) {
            $ans[] = 'FizzBuzz';
        } else if ($i % 3 == 0) {
            $ans[] = 'Fizz';
        } else if ($i % 5 == 0) {
            $ans[] = 'Buzz';
        }
        else {
            $ans[] = (string) $i;
        }
    }
    return $ans;

}

print_r(fizzBuzz(3));

File: Python/Easy/Filter/String/412. Fizz Buzz.py
"""
https://leetcode.com/problems/fizz-buzz/
"""


def fizzBuzz(n: int) -> list[str]:
    result = []
    for i in range(1, n + 1):
        if i % 3 == 0 and i % 5 == 0:
            result.append("FizzBuzz")
        elif i % 3 == 0:
            result.append('Fizz')
        elif i % 5 == 0:
            result.append('Buzz')
        else:
            result.append(f'{i}')
    return result

n = 3
print(fizzBuzz(n))